### DISCUSSION TITLE
(Almost) Monthly Stack-chan community: March '23 / (ほぼ)月刊ｽﾀｯｸﾁｬﾝコミュニティ 2023年3月

### DISCUSSION THREAD

**Discussion by meganetaaan** [meganetaaan/stack-chan#147] [Show and tell]
毎月ｽﾀｯｸﾁｬﾝコミュニティのアップデート情報を GitHub Discussion で残すことにしました。

ChatGPT と連携した「おしゃべりｽﾀｯｸﾁｬﾝ」のツイートをきっかけに、ｽﾀｯｸﾁｬﾝが急速に広まっています！

### ChatGPT 連携

- [きっかけとなった robo8080 さんの作例](https://twitter.com/robo8080/status/1631224076005179393)
  - これ以前にも siri との連携にチャレンジされていました。
- [OpenAI の中の人で日本担当のシェイン・グウ氏が反応する](https://twitter.com/shanegJP/status/1631325117803532288?s=20)
- [ねとらぼが取り上げる](https://nlab.itmedia.co.jp/nl/articles/2303/10/news030.html)
- robo8080さんと紅木タカオさんの貢献によりChatGPT 版ｽﾀｯｸﾁｬﾝが爆発的に広がる
- プロンプトを改造してｽﾀｯｸﾁｬﾝにキャラクター付けするなど改造例が広まる。
  - [執事ロボットに改造する例](https://twitter.com/mijinco_desu/status/1637019015846711297)も！
- いつの間にか `#AIｽﾀｯｸﾁｬﾝ` タグが爆誕
  - [タグ検索結果](https://twitter.com/search?q=%23AI%EF%BD%BD%EF%BE%80%EF%BD%AF%EF%BD%B8%EF%BE%81%EF%BD%AC%EF%BE%9D&src=typed_query&f=live)
- 私も遅ればせながら Moddable 版ファームウェアに ChatGPT 連携を実装しています。
  - [ツイート](https://twitter.com/meganetaaan/status/1632427971729592320?s=20)

### キット化

ChatGPT の反響を受けて、キット化計画を急ピッチで進めています。
近日中に詳しいスケジュールなどを公開できればと思います。

### プルリクエスト

- [M5StackCore2のmanifest\.json設定方法の修正 by Fuji\-P · Pull Request \#143 · meganetaaan/stack\-chan](https://github.com/meganetaaan/stack-chan/pull/143)
- [npm run doctor shows stack\-chan environment info by stc1988 · Pull Request \#142 · meganetaaan/stack\-chan](https://github.com/meganetaaan/stack-chan/pull/142)
- [M5Stack Core2 for AWS専用STLファイルを共有致します by ToshihiroMakuuchi · Pull Request \#137 · meganetaaan/stack\-chan](https://github.com/meganetaaan/stack-chan/pull/137)
- [Update case/contributed/mongonta by mongonta0716 · Pull Request \#141 · meganetaaan/stack\-chan](https://github.com/meganetaaan/stack-chan/pull/141)

- （祝）ｽﾀｯｸﾁｬﾝリポジトリのコントリビュータ（コードやドキュメントを書いて寄贈してくれた人）が 10 人に到達しました！
  - [リポジトリのコントリビュータ一覧](https://github.com/meganetaaan/stack-chan/graphs/contributors)
  - ｽﾀｯｸﾁｬﾝへの貢献はどんな形でも、いつでも大歓迎です！1文字のtypo修正からお気軽にお寄せください。

### 開発

- [robo8080/M5Unified\_StackChan\_ChatGPT: 「ChatGPT API搭載AIｽﾀｯｸﾁｬﾝ」です。](https://github.com/robo8080/M5Unified_StackChan_ChatGPT) （@robo8080 さん）
- [\(1\) 【先行】AIスタックチャンの難所2点（ナレーションなし） \- YouTube](https://www.youtube.com/watch?v=IvXkca5X4JE) （@okimoku1 さん）
  - AIｽﾀｯｸﾁｬﾝを作るための解説動画（製作中）
- ｽﾀｯｸﾁｬﾝの設定用アプリケーションを React 等を使って作成（ @krorcamille さん）
  - [ツイート](https://twitter.com/krorcamille/status/1636554116305674240?s=20)
- Flutter で設定アプリを作成（ @yh1224 さん ）
  - [ツイート](https://twitter.com/yh1224/status/1636705421464600578?s=20)
- ドローン界隈の方にｽﾀｯｸﾁｬﾝが伝わっています！早速さまざまな形で貢献いただき、熱量とスピードに圧倒されますね！
  - SG-90 に代わる良いサーボを調査していただいたり（ @ojiton418 さん）
    - [ツイート](https://twitter.com/ojiton418/status/1638423310571819009)
  - ドローン大会の景品にｽﾀｯｸﾁｬﾝキットが使われたり（報告 @madxfpv さん）
    - [ツイート](https://twitter.com/madxfpv/status/1637645136980242434)
  - ｽﾀｯｸﾁｬﾝを抜き差しできるクレードルを作っていただいたり（ @HS_Glide_path さん）
    - [ツイート](https://twitter.com/HS_Glide_Path/status/1637636928601481217)
- MDF 版ｽﾀｯｸﾁｬﾝを開発中（ @SuspiciousCEO さん）
  - [ツイート](https://twitter.com/SuspiciousCEO/status/1636154578474516481?s=20)

### その他

- ｽﾀｯｸﾁｬﾝにしか見えないセーター（報告 @ClaudeSakuraghi ）
  - [ツイート](https://twitter.com/ClaudeSakuraghi/status/1638730664869793792?s=20)

## ｽﾀｯｸﾁｬﾝ情報をお寄せください！

ChatGPT のバズり以降、タイムラインの流速が速く、ししかわが全てのツイートを拾いきれていません。
上記以外に 3 月に見かけたｽﾀｯｸﾁｬﾝ情報をお持ちでしたらぜひこのスレッドに投稿してください！（自薦他薦は問いません）

**Comment by meganetaaan** [comment]
English translation :pen: 

We have decided to post monthly updates about the StackChan community on GitHub Discussion.

Thanks to the "Talking StackChan" tweet by robo8080, which collaborated with ChatGPT, StackChan has rapidly gained popularity!

### ChatGPT Collaboration

- [Robo8080's example that sparked the collaboration](https://twitter.com/robo8080/status/1631224076005179393)
  - Challenges with Siri were also attempted before this.
- [Shane Guu, the Japanese representative in OpenAI, reacts](https://twitter.com/shanegJP/status/1631325117803532288?s=20)
- [Covered by Net Labo](https://nlab.itmedia.co.jp/nl/articles/2303/10/news030.html)
- Explosive spread of StackChan version with ChatGPT character by robo8080 and Takao Beni's contributions
- Examples of modification, such as modifying the prompt to give StackChan a character, are spreading.
  - [Example of modifying to a butler robot](https://twitter.com/mijinco_desu/status/1637019015846711297)!
- The `#AIStackChan` tag is born unnoticed.
  - [Tag search results](https://twitter.com/search?q=%23AI%EF%BD%BD%EF%BE%80%EF%BD%AF%EF%BD%B8%EF%BE%81%EF%BD%AC%EF%BE%9D&src=typed_query&f=live)
- I have also implemented ChatGPT collaboration into the Moddable version firmware, albeit belatedly.
  - [Tweet](https://twitter.com/meganetaaan/status/1632427971729592320?s=20)

### Kit Production

We are quickly advancing with the plan to make StackChan into a kit, in response to the reaction to ChatGPT. We hope to be able to release detailed schedules and more soon.

### Pull Requests

- [M5StackCore2 manifest.json settings correction by Fuji-P · Pull Request #143 · meganetaaan/stack-chan](https://github.com/meganetaaan/stack-chan/pull/143)
- [npm run doctor shows stack-chan environment info by stc1988 · Pull Request #142 · meganetaaan/stack-chan](https://github.com/meganetaaan/stack-chan/pull/142)
- [We will share M5Stack Core2 for AWS dedicated STL files by ToshihiroMakuuchi · Pull Request #137 · meganetaaan/stack-chan](https://github.com/meganetaaan/stack-chan/pull/137)
- (Congratulations) The StackChan repository contributors (those who wrote code and documentation) have reached 10 people!
  - [List of contributors to the repository](https://github.com/meganetaaan/stack-chan/graphs/contributors)
  - Any contributions to StackChan are welcome, no matter how small, so please feel free to send in any typos or other issues.

### Development

- [robo8080/M5Unified\_StackChan\_ChatGPT: "ChatGPT API-equipped AI Stack-chan"](https://github.com/robo8080/M5Unified_StackChan_ChatGPT) (@robo8080)
- [(1) [Preliminary] Difficult Points of AI Stack-chan (No narration) - YouTube](https://www.youtube.com/watch?v=IvXkca5X4JE) (@okimoku1)
  - Tutorial video for creating an AI Stack-chan (in progress)
- Configuration application for Stack-chan created using React, etc. (@krorcamille)
  - [Tweet](https://twitter.com/krorcamille/status/1636554116305674240?s=20)
- Configuration app created using Flutter (@yh1224)
  - [Tweet](https://twitter.com/yh1224/status/1636705421464600578?s=20)
- Stack-chan has made its way to the drone world! They have already contributed in various ways, and their passion and speed are overwhelming!
  - Investigated good servos to replace SG-90 (@ojiton418)
    - [Tweet](https://twitter.com/ojiton418/status/1638423310571819009)
  - Stack-chan kit used as a prize in a drone competition (@madxfpv)
    - [Tweet](https://twitter.com/madxfpv/status/1637645136980242434)
  - Created a cradle that can be inserted and removed from Stack-chan (@HS_Glide_path)
    - [Tweet](https://twitter.com/HS_Glide_Path/status/1637636928601481217)
- Developing a MDF version of Stack-chan (@SuspiciousCEO)
  - [Tweet](https://twitter.com/SuspiciousCEO/status/1636154578474516481?s=20)

### Miscellaneous

- Sweater that can only be seen by Stack-chan (reported by @ClaudeSakuraghi)
  - [Tweet](https://twitter.com/ClaudeSakuraghi/status/1638730664869793792?s=20)

## Please share information about Stack-chan!

Since the buzz of ChatGPT, the timeline has been moving quickly and @meganetaaan  has not been able to catch every tweet.
If you have any Stack-chan information that you saw in March other than the above, please post it in this thread! (Self-recommendations and recommendations from others are both welcome.)

**Comment by okimoku** [comment]
AIｽﾀｯｸﾁｬﾝに合わせた設定のショートカット（@tie2 さん）
10回ほど会話が続きます。ログも出せるようです。
・[ツイート](https://twitter.com/tie2/status/1638675076630601728?s=20)

AIｽﾀｯｸﾁｬﾝのキャラ設定の簡易ジェネレーター（@coebiten さん）
・[ツイート](https://twitter.com/coebiten/status/1638734244171366400?s=20)
